### ISSUE TITLE
Make omni translate empty OF RSpec

### ISSUE THREAD

**Issue by tcmitchell** [GENI-NSF/geni-tools#10]
If OF has no resources it gives an empty rspec:
```
<rspec><network location="gpo" name="masada"><switches /><links /></network></rspec>
```

But rspec_of refuses to translate that, so you get an Exception in omni.
Fix the rspec_of to handle that.

 _Imported from trac ticket [#10](http://trac.gpolab.bbn.com/gcf/ticket/10),  created by tmitchel on 09-27-2010 at 21:32, last modified: 10-13-2010 at 15:49_

**Comment by ahelsing** [comment]
The problem is that rspec_of.py doesn't recognize the rspec as being an openflow rspec (the can_translate function returns False).

We're adding a pivotal tracker bug to expedient so that the rspec starts with <rspec type="openflow" which is easily recognizable.

Once that is done we can fix this bug.

_Trac comment by jkarlin on 09-29-2010 at 16:28_

**Comment by ahelsing** [comment]
OF Rspec format changed, to include a type="openflow" which rspec_of can match on
-- Included in the 0.3+ version of Expedient

Omni changed to match the OF RSpec format change. This change is included after gcf-1.1


_Trac comment by ahelsing on 10-13-2010 at 15:43_
